cmd/zc_enumerator.go

```diff
@@ -210,6 +213,25 @@ type blobPropsProvider interface {
 	AccessTier() azblob.AccessTierType
 }
 
+func newForcedErrorStoredObject(err, name, relativePath, containerName string) storedObject {
```

Comment:
I see the goal of the change, but I feel a bit uneasy about how we represent the scanning failures. So we are representing each scanning failure as an individual transfer failure, isn't that a bit confusing for regular users?

Reply:
I don't feel like this is confusing for the end user-- If any single unit of a transfer can't be handled, shouldn't it just be a single, tangible error? Do you feel like we should do this for every potential transfer in a container? I feel like that would pollute the job log files and the job plan files unnecessarily.

Alternatively, if you feel that we should not output any failed transfer, In my opinion, in the case of automated users, it makes sense to present an actual failed transfer, because they may depend on `jobs status`.

Is there some alternative thought you have that would satisfy these well?

Reply:
A second note is that this does actively affect the return code of azcopy, unlike logging straight to the scanning log